x-pack/plugins/security_solution/server/lib/sourcerer/routes/index.ts

```diff
@@ -118,3 +255,15 @@ export const createSourcererDataViewRoute = (
     }
   );
 };
+
+const buildDefaultDataview = async (
```

Comment:
can we name this `buildSourcererDataView`? the title of this is misleading because the format should be applied to all kibana data views whether the default or not